presentation-compiler/src/main/dotty/tools/pc/ScalaPresentationCompiler.scala

```diff
@@ -198,6 +199,15 @@ case class ScalaPresentationCompiler(
         .asJava
     }
 
+  def inferExpectedType(params: OffsetParams): CompletableFuture[ju.Optional[String]] =
```

Comment:
There is already an existing method `insertInferredType` wouldn't that be the same thing?

Reply:
This is just for the purpose of testing. The method will be used only inside of pc by such things like completions. This covers more cases then `insertInferredType` but doesn't change it into textEdit. We could have `insertInferredType` cover those cases too but I don't think it's ever useful.